app/views/shared/_banner.html.erb

```diff
@@ -1,4 +1,4 @@
-<div aria-label="banner" role="banner">
+<header aria-label="banner">
```

Comment:
This label isn't translated. Stepping back a bit, I wonder why we have so many different banners/headers in the first place. A few lines below we use a `header` for the "official website" banner, so we would want unique labels if we were to have multiple `header`s. We could go that route if the headers are meaningfully different, or maybe we could eliminate one or the other.

Reply:
It sounds like it would be a good idea to translate all the `aria-label` attributes to be translatable. I can make a separate ticket to make sure that all aria-labels are translatable.

I think we should remove that second (or original) header. If we want to stick to the rules of a `header` element, it should be used for introductory content i.e a group of introductory or navigational aids ([source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)) for which the blue banner and navigation apply. Also, `<header>` should not be nested inside of a `<section>`

Reply:
> I think we should remove that second (or original) header.

Yeah, that could work. If I had to guess, the markup for the inner "official USA website" banner was likely copied verbatim from the [design system documentation](https://designsystem.digital.gov/components/banner/), but I don't know that the header _must_ be a `header` element, if it makes more sense that the header also contains our "no PII" banner.